### PR TITLE
Updated elephpants

### PR DESCRIPTION
**Pull request by IgorSantos17**
- Updated some elephpants infos

### CODE REVIEW

File: resources/data/elephpants.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
 		{
             "id": 7,
 			"name": "Chili",
-			"description": "Zend Red, ZendCon 2013",
+			"description": "ZendCon 2013",
```
Comment (jgrossi) [inline resources/data/elephpants.json]: some people also call it "zend red" one

File: resources/data/elephpants.json [hunk 2]
```diff
@@ -148,14 +148,14 @@
...
-			"sponsor": "AmsterdamPHP",
+			"sponsor": "AmsterdamPHP Community",
 			"year": 2014,
 			"image": "https://i.imgur.com/GdFxJmk.jpg"
 		},
 		{
             "id": 20,
 			"name": "Phil Snow",
-			"description": "ConFoo",
+			"description": "ConFoo web techno conference",
```
Comment (jgrossi) [inline resources/data/elephpants.json]: please, fix caps here :-) or is it the right way to write?

File: resources/data/elephpants.json [hunk 3]
```diff
@@ -359,7 +359,14 @@
...
             "year": 2019,
             "image": "https://i.imgur.com/QIyckVs.jpg"
+        },
+        {
+            "id": 46,
+            "name": "CakeDC Fest",
+            "description": "CakeDC Fest",
+            "sponsor": "CakePHP",
+            "year": 2017,
+            "image": "https://live.staticflickr.com/65535/48796213323_1efccd304c_n.jpg"
```
Comment (jgrossi) [inline resources/data/elephpants.json]: please crop the image as a square to get the best shoot for the elePHPant. our command to add elePHPants will fit it in a square from the center, and I guess it's gonna cut his head :D

File: resources/data/elephpants.json [hunk 4]
```diff
@@ -227,15 +227,15 @@
...
+			"description": "CakePHP Framework",
 			"sponsor": "CakePHP",
 			"year": 2017,
 			"image": "https://i.imgur.com/CKkne1L.jpg"
 		},
 		{
             "id": 30,
-			"name": "Cake Foundation",
-			"description": "CakeSF",
+			"name": "CakeSF",
```
Comment (jgrossi) [inline resources/data/elephpants.json]: I think Cake Software Foundation is the right name, can you double check please?

File: resources/data/elephpants.json [hunk 5]
```diff
@@ -316,7 +316,7 @@
             "id": 40,
             "name": "PHPCE",
             "description": "PHPCE 2018",
-            "sponsor": "PHPCE",
```
Comment (jgrossi) [inline resources/data/elephpants.json]: phpce is an annual conference

File: resources/data/elephpants.json (hunk 1 above)
Comment (IgorSantos17) [reply resources/data/elephpants.json]: I didn't know, reversed

File: resources/data/elephpants.json (hunk 2 above)
Comment (IgorSantos17) [reply resources/data/elephpants.json]: adjusted

File: resources/data/elephpants.json (hunk 4 above)
Comment (IgorSantos17) [reply resources/data/elephpants.json]: Cake Software Foundation = CakeSF
and in description all name: Cake Software Foundation.
do you agree?

File: resources/data/elephpants.json (hunk 5 above)
Comment (IgorSantos17) [reply resources/data/elephpants.json]: adjusted

File: resources/data/elephpants.json (hunk 4 above)
Comment (jgrossi) [reply resources/data/elephpants.json]: sounds good

File: resources/data/elephpants.json (hunk 3 above)
Comment (cookieguru) [reply resources/data/elephpants.json]: This photo isn't the CakeFest elePHPant

File: resources/data/elephpants.json [hunk 6]
```diff
@@ -359,7 +359,14 @@
             "sponsor": "PHP Conference Poland",
             "year": 2019,
             "image": "https://i.imgur.com/QIyckVs.jpg"
+        },
+        {
+            "id": 46,
+            "name": "CakeDC Fest",
```
Comment (cookieguru) [inline resources/data/elephpants.json]: It's just `CakeFest`

Comment (jgrossi) [reply resources/data/elephpants.json]: I think this one is just `CakeDC`, correct?

File: resources/data/elephpants.json [hunk 7]
```diff
@@ -359,7 +359,14 @@
...
+            "description": "CakeDC Fest",
```
Comment (jgrossi) [inline resources/data/elephpants.json]: `Cake Development Corporation`

File: resources/data/elephpants.json [hunk 8]
```diff
@@ -359,7 +359,14 @@
...
+            "sponsor": "CakePHP",
```
Comment (jgrossi) [inline resources/data/elephpants.json]: `CakeDC`

File: resources/data/elephpants.json (hunk 6 above)
Comment (cookieguru) [reply resources/data/elephpants.json]: Yes:
CakeDC photo

File: resources/data/elephpants.json (hunk 8 above)
Comment (IgorSantos17) [reply resources/data/elephpants.json]: adjusted

File: resources/data/elephpants.json (hunk 7 above)
Comment (IgorSantos17) [reply resources/data/elephpants.json]: adjusted

**Comment by IgorSantos17** [comment]
Hey @cookieguru 
I will make a new PR by adding these Cake elephpants and some missing ones including the adjusted photos.
So I decided to remove.

Ps: We even talked a little bit about it in the issue: #16

**Pull request merged**